### PR TITLE
Scoda qa week2

### PR DESCRIPTION
**Pull request by PhakamaniXulu**
- Scoda QA

### CODE REVIEW

**Comment by PhakamaniXulu** [comment]
@callum09 can you please review. Thanks

File: scoda/static/src/sass/components/_about-us.scss [hunk 1]
```diff
@@ -16,9 +16,7 @@
         line-height: 28px;
         font-weight: 400;
     }
-    img{
-        width: 100%;
-    }
+
```
Comment (callum09) [inline scoda/static/src/sass/components/_about-us.scss:19]: @PhakamaniXulu the spacing and indentation in this file is not good at all. Please leave code better than you found it.

File: scoda/static/src/sass/components/_cta_block.scss [hunk 2]
```diff
@@ -84,12 +85,13 @@
         background-color: rgba(255,255,255,0.5);
         box-shadow: 0 0 6px 2px rgba(0,0,0,0.2);
         margin-bottom: 50px;
+        width:Calc(60%);
```
Comment (callum09) [inline scoda/static/src/sass/components/_cta_block.scss:105]: @PhakamaniXulu no need for Calc if there's no calculation.

Comment (callum09) [reply scoda/static/src/sass/components/_cta_block.scss:105]: Please resolve spacing and indentation

File: scoda/static/src/sass/components/_cta_block.scss [hunk 3]
```diff
@@ -167,24 +172,80 @@
...
                 padding-bottom: 50px;
             }
         }
         .cta-info{
+            
             .cta-image{
                 width: 100% !important;
                 background-size: 100% 100%;
                 margin-bottom: 30px;
+                margin-left:0px;
```
Comment (callum09) [inline scoda/static/src/sass/components/_cta_block.scss:285]: Please use best practise for spacing declarations, properties and nesting:

```
.cta {
 &-image {
   margin-left: 0;
  }
 &-info { ... }
}
```
Note that 0px is not used, just 0. There should be space after the property like `margin-left: 0`, not `margin-left:0;`

Comment (callum09) [reply scoda/static/src/sass/components/_cta_block.scss:285]: @PhakamaniXulu

File: scoda/static/src/sass/components/_datastories.scss [hunk 4]
```diff
@@ -62,6 +62,7 @@ body {
 .ds-banner-logo {
```
Comment (callum09) [inline scoda/static/src/sass/components/_datastories.scss:65]: @PhakamaniXulu please review spacing and indentation here. Use beautify or pretty print to format.

**Pull request merged**